dask/dataframe/rolling.py

```diff
@@ -110,10 +113,15 @@ def _call_method(self, method_name, *args, **kwargs):
             self.rolling_kwargs, method_name, args, kwargs)}
         if self.rolling_kwargs['axis'] in [0, 'rows']:
             # roll in the partition direction (will need to access neighbor)
+            window = self.rolling_kwargs['window']
+            tail_name = '{}-tailprev-{}'.format(old_name, window-1)
```

Comment:
This isn't essential for correctness, but will help out the diagnostics
that watch and summarize execution.  This is just a convention.

On Thu, Jun 9, 2016 at 7:40 PM, Matthew Rocklin <mrocklin@continuum.io>
wrote:

> The goal is to ensure that LONGHASHSTRING is at the end of the name, not
> in the middle.
>
> On Thu, Jun 9, 2016 at 7:39 PM, Mike Graham <notifications@github.com>
> wrote:
>
>> In dask/dataframe/rolling.py
>> <https://github.com/dask/dask/pull/1270#discussion_r66554809>:
>>
>> > @@ -110,10 +113,15 @@ def _call_method(self, method_name, *args, **kwargs):
>> >              self.rolling_kwargs, method_name, args, kwargs)}
>> >          if self.rolling_kwargs['axis'] in [0, 'rows']:
>> >              # roll in the partition direction (will need to access neighbor)
>> > +            window = self.rolling_kwargs['window']
>> > +            tail_name = '{}-tailprev-{}'.format(old_name, window-1)
>>
>> old_name should already have some LONGHASHSTRING. Is it essential to add
>> more?
>>
>> —
>> You are receiving this because you commented.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/dask/dask/pull/1270/files/dcfd39d14c47185a16bd05d79a97394b78183738#r66554809>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/AASszGaBd-yV8QGOreoPwXYN8fNVBaHhks5qKM5mgaJpZM4Iyfh5>
>> .
>>
>
>